src/emulator/auth/operations.ts

```diff
@@ -1185,8 +1181,22 @@ export function setAccountInfoImpl(
     }
   }
 
+  if (reqBody.linkProviderUserInfo) {
```

Comment:
Thanks for the PR! Please add a test to `src/test/emulators/auth/setAccountInfo.spec.ts` that covers the changes. The changes otherwise look great. Let me know if you need any help